webpack.config.production.js

```diff
@@ -1,23 +1,23 @@
 /* eslint-disable */
 
-var path = require("path");
-var webpack = require("webpack");
+var path = require('path');
```

Comment:
As a helpful tip, take a look at how I linted a Spectacle presentation in https://github.com/ryan-roemer/surge2015/blob/master/package.json#L10-L12

Side note -- the lint may be too unwieldy / complicate this specific PR, so feel free to ticket out + revert the quotes thing and then have this PR just focus on the babel stuff.

Thanks!